app/deploy/generator/04-syndesis-verifier.yml.mustache

```diff
@@ -90,6 +90,10 @@
           volumeMounts:
           - name: config-volume
             mountPath: /deployments/config
+        initContainers:
+        - name: init-syndesis-db
+          image: busybox
+          command: ['sh', '-c', 'until nslookup syndesis-db; do echo waiting for service; sleep 2; done;']
```

Comment:
Actually I wouldn't want the verifier to depend on the DB only because a sample connection uses this DB. I see it more likely that we forget to remove this than that someone hits the "validate" button on the UI before the DB starts. And if so, then the validation fails, which is fine (that is what validation is for after all)

Other than that the verifier doesnt use the db at all.